src/imitation/policies/serialize.py

```diff
@@ -2,6 +2,9 @@
 # TODO(scottemmons): Only import packages and modules to adhere to style guide:
 #  https://google.github.io/styleguide/pyguide.html#22-imports
 
+# FIXME(sam): it seems like this module could mostly be replaced with a few
```

Comment:
Investigate this, but don't go too deep (would be nice to simplify, but it's not a high-value change).